verify_rpms/rpm_verifier.py

```diff
@@ -233,6 +233,30 @@ def aggregate_results(processed_image_list: list[ProcessedImage]) -> dict[str, A
     return aggregated_results
 
 
+def generate_images_processed_result(
```

Comment:
Looking at the description below, I think this will be a clearer name:
```suggestion
def generate_processed_image_digests(
```

Reply:
Done